sct-commons/pom.xml

```diff
@@ -74,19 +74,19 @@
         <dependency>
```

Comment:
maybe replace these dependancies by : 
```xml

<!--mockito-bom is Mockito's Bill Of Materials (BOM). It will ensure to align and manage all Mockito dependency versions for you.-->
<dependency>
    <groupId>org.mockito</groupId>
    <artifactId>mockito-bom</artifactId>
    <version>4.9.0</version>
    <type>pom</type>
</dependency>

```

Reply:
I prefer to update version and centralise it. Because bom import 6 dependencies and we just need 3 of them.